be/src/util/reusable_closure.h

```diff
@@ -0,0 +1,61 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include <google/protobuf/stubs/common.h>
+
+#include <atomic>
+
+#include "service/brpc.h"
+
+namespace starrocks {
+
+template <typename T>
+class ReusableClosure : public google::protobuf::Closure {
```

Comment:
It has already been defined in exec/tablet_sink.h

Reply:
remove from it